### PR TITLE
Task/make successful request for reg endpoint

### PR DESCRIPTION
**Pull request by marckraw**
Registration endpoint works fine, there is snackbar with information about success after registration or not.

### CODE REVIEW

File: frontend/src/services/auth.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import * as axios from 'axios';
 
-
 axios.defaults.baseUrl = process.env.VUE_APP_API_LOCAL_URL;
 
+const apiUrl = process.env.VUE_APP_API_LOCAL_URL;
```
Comment (BartoszBaczek) [inline frontend/src/services/auth.js]: do we need both this:
`  axios.defaults.baseUrl = process.env.VUE_APP_API_LOCAL_URL; `
and this:
 `const apiUrl = process.env.VUE_APP_API_LOCAL_URL; `
?

File: frontend/src/services/auth.js [hunk 2]
```diff
@@ -18,21 +19,9 @@ const authService = {
     },
 
     register(userData) {
-        console.log('user data from auth.register method');
         console.dir(userData);
```
Comment (BartoszBaczek) [inline frontend/src/services/auth.js]: remove plz

File: frontend/src/store/authModule.js [hunk 3]
```diff
@@ -1,25 +1,54 @@
...
         return state.authenticationError !== '';
     },
 };
 
 const mutations = {
+    ADD_REGISTER_ERROR(state, error) {
+        state.registerError = error.response.statusText;
+    },
+    ADD_REGISTER_SUCCES_MESSAGES(state, data) {
+        state.registerSnackBarColor = 'success';
```
Comment (BartoszBaczek) [inline frontend/src/store/authModule.js]: imo it should be handled by css

File: frontend/src/store/authModule.js [hunk 4]
```diff
@@ -1,25 +1,54 @@
...
+    registerSuccessMessage: '',
+    registerSnackBarColor: 'error',
     isAuthenticated: false,
     authenticationError: '',
 };
 
 const getters = {
-    isAuthenticated(state) {
+    registerSnackBarColor() {
+        return state.registerSnackBarColor;
```
Comment (BartoszBaczek) [inline frontend/src/store/authModule.js]: imo it should be handled by css

File: frontend/src/store/authModule.js [hunk 5]
```diff
@@ -1,25 +1,54 @@
 import auth from '../services/auth';
 
 const state = {
+    registerErrorMessage: '',
+    registerSuccessMessage: '',
+    registerSnackBarColor: 'error',
```
Comment (BartoszBaczek) [inline frontend/src/store/authModule.js]: imo it should be handled by css

File: frontend/src/store/authModule.js [hunk 6]
```diff
@@ -1,25 +1,54 @@
...
+    CLEAR_REGISTER_MESSAGES(state) {
+        state.registerErrorMessage = '';
+        state.registerSuccessMessage = '';
+    },
     AUTHENTICATE(state) {
         state.isAuthenticated = true;
     },
     ADD_AUTHENTICATION_ERROR(state, error) {
-        state.authenticationError = error;
+        state.authenticationError = 'asdasd';
```
Comment (BartoszBaczek) [inline frontend/src/store/authModule.js]: IMO not very descriptive error message

File: frontend/src/services/auth.js (hunk 1 above)
Comment (marckraw) [inline frontend/src/services/auth.js]: We will use that approach for now instead `axios.defaults.baseUrl = process.env.VUE_APP_API_LOCAL_URL;` cause it's just not working for some reason.

Comment (marckraw) [reply frontend/src/services/auth.js]: Yup, I will remove the `axios.defaults.baseUrl` for now.

File: frontend/src/services/auth.js (hunk 2 above)
Comment (marckraw) [reply frontend/src/services/auth.js]: sure thing.

File: frontend/src/store/authModule.js (hunk 6 above)
Comment (marckraw) [reply frontend/src/store/authModule.js]: yup, forget to push changes - -' will do.

File: frontend/src/store/authModule.js (hunk 5 above)
Comment (marckraw) [reply frontend/src/store/authModule.js]: this is for the snackbar comoponent, which want js variable i guess

**Pull request merged**